docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
 `Flathub <https://flathub.org/>`_, can have restrictions in this respect).
 
 The only technical requirements made by Flatpak are that applications follow a
-small number of Freedesktop standards, in order to enable desktop integration
+small number of FreeDesktop standards, in order to enable desktop integration
 (see :doc:`conventions`).
 
+Issues of current model of packaging
+------------------------------------
+
+It is important to understand the problems of the current model
+of packaging applications to understand the existence of Flatpak:
+
+- **Duplicated work packaging apps**: Many Linux distributions come with their own package
+  manager, package format and repository. This requires a lot of maintainers to package the
+  same application in various distributions, and/or the application developer to learn the
+  language of each format and then package the application in those distributions, and/or
+  ignore most distributions and package and support a couple of distributions. This makes
+  it very difficult to innovate the Linux desktop as a whole as a considerably high amount of time
+  is spent on packaging.
+- **Limited to apps that are packaged**: Not all applications are natively available
+  in every Linux distribution. If an application is not available in a specific
+  distribution, the user will have to rely on manually downloading the archive
+  of the application, extracting it and hoping the application will launch.
+- **Limited to distributions that have the apps**: The user is limited to the
+  number of distributions that have the needed applications for them
+  to properly setup their workflow. This reduces the amount of distributions
+  that can be suitable for a user.
+- **Hard to innovate in OS space**: The maintainers of the distributions have to spend a lot of
+  time packaging applications to make the distribution suitable for the end user, instead of focusing
+  on their end goals. This delays the progress of each distribution.
```

Comment:
I think we need some mention of the lack of sandboxing in the current packaging formats. For example deb-packaged applications have full filesystem access, full access to D-Bus, etc.

Reply:
Also worth mentioning that LTS distributions often have very old versions of applications packaged natively, bug reproducibility is hindered by the different environments that applications are run in, and application developers often have little control over how their application is packaged by distributions.

Reply:
> I think we need some mention of the lack of sandboxing in the current packaging formats. For example deb-packaged applications have full filesystem access, full access to D-Bus, etc.

I was planning to write that, but knowing the current situation of Flatpak's sandboxing, I think it's best to not mention it yet.

Reply:
> > I think we need some mention of the lack of sandboxing in the current packaging formats. For example deb-packaged applications have full filesystem access, full access to D-Bus, etc.
> 
> I was planning to write that, but knowing the current situation of Flatpak's sandboxing with static permissions, I think it's best to not mention it yet. I think it's best to wait for dynamic permissions to become more used.

I disagree. Sandboxing is one of Flatpak's main features, and pointing out that traditional packaging formats don't have it at all doesn't imply the status quo with Flatpak apps is excellent.

Reply:
Sure, I will add it soon.